unittests.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
+from __future__ import print_function, absolute_import
```

Comment:
Is `print_function` needed here? And what is `absolute_import` needed for?

Reply:
imports like `import .textimporter` were used like `import textimporter` in Python 2. So it would look for local modules first, then for packages in the pythonpath. In python 3 this ambiguity was resolved and `import .textimporter` imports a local module, while `import textimporter` tries to import a package. `absolute_import` to maintain backwards compatibility. 
`print_function` is not needed here. We could think of a general wild import from `__future__` in all files?

Reply:
It's a bit annyoing if you just want to print something and always need to add that statement first... on the other hand - the committed code should not contain `print` anymore. And python 2.7 eats `print()` just fine

Reply:
Maybe a wild import is best then? I agree that it is annoying to always have to include `print_function`. 

In my setup, I'm always running tests agains Python 3 locally, and there is no need for `print_function`... 

Thinking about it, I think we can leave it as is. One redundant import from future won't hurt. Closing this conversation.